docs/Agent/user-guide/man_page.md

```diff
@@ -1 +1,439 @@
-# Man Page
+# Man pages
+
+---
+
+### pbench-clear-results
+
+---
+
+**NAME**  
+pbench-clear-results, Clears the result directory
+
+**SYNOPSIS**  
+`pbench-clear-results [-C][--help]`
+
+**DESCRIPTION**  
+This command clears the results directory from /var/lib/pbench-agent directory
+
+**OPTIONS**  
+ -C, --config PATH  
+Path to a pbench-agent configuration file (defaults to the '\_PBENCH_AGENT_CONFIG' environment variable, if defined) [required]
+
+--help  
+Show this message and exit.
+
+---
+
+### pbench-clear-tools
+
+---
+
+**NAME**  
+pbench-clear-tools, Clear registered tools by name or group
+
+**SYNOPSIS**  
+`pbench-clear-tools [-C][-n][-g][-r][--help]
+`
+
+**DESCRIPTION**  
+Clear all tools which are registered and can filter by name of the group
+
+**OPTIONS**  
+-C, --config PATH  
+Path to a pbench-agent configuration file (defaults to the '\_PBENCH_AGENT_CONFIG' environment variable, if defined) [required]
+
+-n, --name,--names STR  
+Clear only the <tool-name> tool.
+
+-g, --group,--groups STR  
+Clear the tools in the <group-name> group. If no group is specified, the 'default' group is assumed.
+
+-r, --remote, --remotes STR  
+Clear the tool(s) only on the specified remote(s). Multiple remotes may be specified as a comma-separated list. If no remote is specified, all remotes are cleared
+
+--help  
+Show this message and exit.
+
+---
+
+### pbench-copy-results
```

Comment:
I think it's important to describe somewhere the difference between the `pbench-*-results` tools and the `pbench-results-*` tools. The `pbench-*-results` versions are used only with a `/opt/pbench-agent/id_rsa` file to use the `scp` protocol to push a tarball to a 0.69 Pbench Server ... that is, the "pass-through" server or the AWS Pbench Satellite server. The `pbench-results-*` versions are used only for a 1.0 Pbench Server, with an API Key authentication token rather than a private `id_rsa` file.

It's actually really confusing to have both, especially if they're not clearly described. But the idea is that we'll begin to phase out the 0.69 `pbench-*-results` variants. I expect they'll still be in Pbench Agent 0.73, but gone by Pbench Agent 1.0.

Reply:
I'm not certain that we want to describe the v0.69 vs. v1.0 difference in the individual sections.  I think that would be better represented in a summary/ToC section.  (But, I agree we should have such an explanation prominently displayed.)

This does bring up a question as to how the user of this and other commands is supposed to know about all the things which the command implicitly requires:
- It needs the Agent config file (normally set up during Agent installation and configured automatically during login).
- It needs an SSH key, an access token, or an API key (depending on the command and use)
- etc....